src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -0,0 +1,263 @@
+        return connected
+
+    @AbstractFocuser.position.getter
+    def position(self):
+        """
+        Returns current focus position in the lens focus encoder units.
+        """
+        response = self._send_command('P#', response_length=1)[0].replace("#", "")
+        return response
+
+    @property
+    def min_position(self):
+        """
+        Returns position of close limit of focus travel, in encoder units.
+        """
+        return None
+
+    @property
+    def max_position(self):
+        """
+        Returns position of far limit of focus travel, in encoder units.
+        """
+        return None
+
+    @property
+    def is_moving(self):
+        """ True if the focuser is currently moving. """
+        return self._is_moving
+
+    ##################################################################################################
+    # Public Methods
+    ##################################################################################################
+
+    def connect(self, port):
+        try:
+            # Configure serial port.
+            self._serial_port = serial.Serial()
+            self._serial_port.port = port
+            self._serial_port.baudrate = 38400
+            self._serial_port.bytesize = serial.EIGHTBITS
+            self._serial_port.parity = serial.PARITY_NONE
+            self._serial_port.stopbits = serial.STOPBITS_ONE
+            self._serial_port.timeout = 2.0
+            self._serial_port.xonxoff = False
+            self._serial_port.rtscts = False
+            self._serial_port.dsrdtr = False
+            self._serial_port.write_timeout = None
+            self._inter_byte_timeout = None
+
+            # Establish connection
+            self._serial_port.open()
+
+        except serial.SerialException as err:
+            self._serial_port = None
+            self.logger.critical('Could not open {}!'.format(port))
+            raise err
+
+        # Want to use a io.TextWrapper in order to have a readline() method with universal newlines.
+        # The line_buffering option causes an automatic flush() when
+        # a write contains a newline character.
+        self._serial_io = io.TextIOWrapper(io.BufferedRWPair(self._serial_port, self._serial_port),
+                                           newline='\n', encoding='ascii', line_buffering=True)
+        self.logger.debug('Established serial connection to {} on {}.'.format(self.name, port))
+
+    def move_to(self, new_position):
+        """
+        Moves focuser to a new position.
+
+        Args:
+            position (int): new focuser position, in encoder units
+
+        Returns:
+            int: focuser position following the move, in encoder units.
+
+        Does not do any checking of the requested position but will warn if the lens reports
+        hitting a stop.
+        """
+        self._is_moving = True
+        try:
+            self._send_command('M{:d}#'.format(int(new_position)), response_length=0)
```

Comment:
@JAAlvarado-Montes, I think the f-strings are a lot cleaner in most places and would encourage you to use them as the default, but especially in something like this I think the extra `format` call makes it harder to see the command being written.

```suggestion
            self._send_command(f'M{int(new_position):d}#'), response_length=0)
```